contentcuration/contentcuration/viewsets/user.py

```diff
@@ -39,6 +39,19 @@
 from contentcuration.viewsets.sync.constants import VIEWER_M2M
 
 
+class IsAdminUser(BasePermission):
+    """
+    Our custom permission to check admin authorization.
+    """
+
+    def has_permission(self, request, view):
+        try:
+            perm = request.user and request.user.is_admin
```

Comment:
Can just return early here (and don't think you need to coerce to bool)
```suggestion
            return request.user and request.user.is_admin
```